cmd/manager/main.go

```diff
@@ -103,6 +103,13 @@ func newRootCommand() *cobra.Command {
 				log.WithError(err).Fatal("Cannot parse log level")
 			}
 			log.SetLevel(level)
+
+			// Add some millisecond precision to log timestamps, useful for debugging performance.
```

Comment:
Do we want the millisecond precision all the time, or just when we are debugging performance?

Reply:
It's not critical but I felt it was probably useful and harmless to keep.

Reply:
Sounds good.